<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="temprature">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>841</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>home</string>
    </attribute>
    <widget class="QLabel" name="MUNCHCULTURE">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>441</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; color:#845f64;&quot;&gt;MUNCH CULTURE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>GESTION DES EMPLOYEE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>280</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>GESTION PROFILS</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Gestion des employee</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>781</width>
       <height>621</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">tabWidget_2 = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100)</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajout</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date_emb</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id_employee</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>salaire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>220</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="salaire">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="type">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>210</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Ajouter">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="date_emb">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>270</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>280</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>340</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>330</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>410</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mdp</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mdp">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>400</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Affichage</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>471</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_recherche">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>50</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Id_employee</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>date naissance employee</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>adressse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>numero de tel </string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="le_recherche">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_supprimer">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>140</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>id_stock</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>400</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date naiss employee</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="num_com_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>370</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>460</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numero tel </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_36">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>520</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date d'arrivee</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>370</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id_employee</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="quantite_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>430</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_38">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>430</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse </string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>430</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="colone_tri">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Id_employee</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>date de naissance employee</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>adresse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>num de tel</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="ordre_tri">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>90</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>asc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>desc</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="button_tri">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>mailing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PDF">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="produit_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>400</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="date_arrivee_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>520</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="quantite_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>460</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestion de profil</string>
    </attribute>
    <widget class="QTabWidget" name="affichage2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>771</width>
       <height>601</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Ajout</string>
      </attribute>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>login</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_profil</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mdp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numero</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_profil">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="mdp_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter1">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>Affichage</string>
      </attribute>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>441</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_recherche_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>id_profil</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>date de naissance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>adresse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>numero de tel</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="le_recherche_2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QComboBox" name="colone_tri_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>id_profil</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>date de naissance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>adresse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>numero de tel</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="ordre_tri_2">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>70</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>ASC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DESC</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="button_tri_2">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_2">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>130</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>mailing</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_supprimer_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>130</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>id_f</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_39">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>300</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_profil</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>330</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date de naissance</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_41">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>360</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_42">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>390</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numero de tel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="numventemodif">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>300</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="quantmodif">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>360</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="idmodif">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>390</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>370</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prod_modif">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>330</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>197</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
